### ISSUE TITLE
formData not supported

### ISSUE THREAD

**Issue by ryanreich** [iriscouch/browser-request#63]

**Comment by edenpessach** [comment]
isn't [this](https://github.com/iriscouch/browser-request/blob/master/index.js#L125) what you are looking for?

try using 'form' instead of 'body'

**Comment by qvaqvaboo** [comment]
And make sure your method is 'POST', not 'post' (capital letters)
https://github.com/iriscouch/browser-request/blob/master/index.js#L127

**Comment by Ajk4** [comment]
I think OP meant exactly 'formData' parameter available in node. See ( #https://www.npmjs.com/package/request#requestoptions-callback )

I believe it is needed if you want to send files using multipart forms.

**Comment by frxncisjoseph** [comment]
@Ajk4 Indeed, nothing has been mentioned about this nor a response. This truly is something that is crucial...
